R/layers-convolutional.R

```diff
@@ -841,12 +841,13 @@ layer_conv_lstm_2d <- function(object, filters, kernel_size, strides = c(1L, 1L)
                                unit_forget_bias = TRUE, kernel_regularizer = NULL, recurrent_regularizer = NULL, bias_regularizer = NULL, 
                                activity_regularizer = NULL, kernel_constraint = NULL, recurrent_constraint = NULL, bias_constraint = NULL, 
                                return_sequences = FALSE, go_backwards = FALSE, stateful = FALSE, dropout = 0.0, recurrent_dropout = 0.0,
-                               batch_size = NULL, name = NULL, trainable = NULL, weights = NULL) {
+                               batch_size = NULL, name = NULL, trainable = NULL, weights = NULL, input_shape = NULL) {
```

Comment:
@jjallaire I added this since input_shape wasn't an argument to `layer_conv_lstm`.